### PR TITLE
Suggestion to make the pull request template more "concise"

### PR DESCRIPTION
**Pull request by MaEtUgR**
## Solved Problem
I see lots of newcomers straight ignoring the pull request template.
Many times the whole

> Please use [PX4 Discuss](http://discuss.px4.io/) or [Discord](https://discord.gg/dronecode) to align on pull requests if necessary. You can then open draft pull requests to get early feedback.

is still in and probably nothing was read.

## Solution
I suggest keeping it short, no full text, the top part commented.

## Context
image

### CODE REVIEW

**Comment by mrpollo** [comment]
I like the changes, perhaps the biggest problem is policing the template is being used.

**Comment by MaEtUgR** [comment]
> the biggest problem is policing the template is being used

I think policing needs to ensure good descriptions not following the template. The template should simplify creating a pull request description that's living up to expectations hence my suggestion. That doesn't mean a contributor can't make his own structure and not follow the template. That description can be adapted to the particular change and equally or even more useful.

**Comment by MaEtUgR** [comment]
Any feedback on this? I'm happy to iterate.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,17 +1,29 @@
...
-## Describe your solution
-A clear and concise description of what you have implemented.
+Get early feedback through
+- Dronecode Discord: https://discord.gg/dronecode
+- PX4 Discuss: http://discuss.px4.io/
+- opening a draft pr and sharing the link
 
-## Describe possible alternatives
-A clear and concise description of alternative solutions or features you've considered.
+-->
```
Comment (junwoo091400) [inline .github/PULL_REQUEST_TEMPLATE.md:10]: We could add something like this: (anything that reminds users to upload the ULog!! As daniel said during the dev call.

```
If you have a log, make sure to upload it to https://review.px4.io/ (it accelerates review / feedback process)
```

**Review by junwoo091400 — APPROVED** [review]
In general seems like a good change!! :tada:

**Comment by MaEtUgR** [comment]
I guess if there are no more opinions then I'll merge this, wait for further feedback and we can iterate e.g. on making log upload even more prominent.

**Pull request merged**